### PR TITLE
Added check connectivity in device

### PR DESCRIPTION
**Pull request by vvlladd28**
## Pull Request description

A new dialog has been added that helps to check the device's connectivity.

This dialog is displayed by default after the device is created, and there is an option don't to show it anymore:
image

In the dialogue, you can see the telemetry that was sent:
image

Also, this dialogue can be called for the created device:
image
image

## General checklist

- [x] You have reviewed the guidelines [document](https://docs.google.com/document/d/1wqcOafLx5hth8SAg4dqV_LV3un3m5WYR8RdTJ4MbbUM/edit?usp=sharing).
- [x] [Labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#about-labels) that classify your pull request have been added.
- [x] The [milestone](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones) is specified and corresponds to fix version.  
- [ ] Description references specific [issue](https://github.com/thingsboard/thingsboard/issues).
- [x] Description contains human-readable scope of changes.
- [x] Description contains brief notes about what needs to be added to the documentation.
- [x] No merge conflicts, commented blocks of code, code formatting issues.
- [x] Changes are backward compatible or upgrade script is provided.
- [x] Similar PR is opened for PE version to simplify merge. Crosslinks between PRs added. Required for internal contributors only.
  
## Front-End feature checklist

- [x] Screenshots with affected component(s) are added. The best option is to provide 2 screens: before and after changes;
- [x] If you change the widget or other API, ensure it is backward-compatible or upgrade script is present.
- [ ] Ensure new API is documented [here](https://github.com/thingsboard/thingsboard-ui-help)

## Back-End feature checklist

- [ ] Added corresponding unit and/or integration test(s). Provide written explanation in the PR description if you have failed to add tests.
- [ ] If new dependency was added: the dependency tree is checked for conflicts.
- [ ] If new service was added: the service is marked with corresponding @TbCoreComponent, @TbRuleEngineComponent, @TbTransportComponent, etc.
- [ ] If new REST API was added: the RestClient.java was updated, issue for [Python REST client](https://github.com/thingsboard/thingsboard-python-rest-client) is created.

### CODE REVIEW

File: application/src/main/java/org/thingsboard/server/controller/BaseController.java [hunk 1]
```diff
@@ -208,6 +209,9 @@ public abstract class BaseController {
     @Autowired
     protected DeviceService deviceService;
 
+    @Autowired
```
Comment (ashvayka) [inline application/src/main/java/org/thingsboard/server/controller/BaseController.java:212]: No need to add this to the base controller. Add only to the DeviceConnectivityController.

File: application/src/main/java/org/thingsboard/server/controller/DeviceConnectivityController.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                                                      @PathVariable(DEVICE_ID) String strDeviceId, HttpServletRequest request) throws ThingsboardException, URISyntaxException {
+        checkParameter(DEVICE_ID, strDeviceId);
+        DeviceId deviceId = new DeviceId(toUUID(strDeviceId));
+        Device device = checkDeviceId(deviceId, Operation.READ_CREDENTIALS);
+
+        String baseUrl = systemSecurityService.getBaseUrl(getTenantId(), getCurrentUser().getCustomerId(), request);
+        return deviceConnectivityService.findDevicePublishTelemetryCommands(baseUrl, device);
+    }
+
+    @ApiOperation(value = "Download mqtt ssl certificate using file path defined in device.connectivity properties (downloadMqttServerCertificate)", notes = "Download Mqtt server certificate." + SYSTEM_OR_TENANT_AUTHORITY_PARAGRAPH)
```
Comment (ashvayka) [inline application/src/main/java/org/thingsboard/server/controller/DeviceConnectivityController.java:89]: SYSTEM_OR_TENANT_AUTHORITY_PARAGRAPH is not needed.

File: application/src/main/java/org/thingsboard/server/controller/DeviceConnectivityController.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    @RequestMapping(value = "/device-connectivity/{protocol}/certificate/download", method = RequestMethod.GET)
+    @ResponseBody
+    public ResponseEntity<org.springframework.core.io.Resource> downloadMqttServerCertificate(@ApiParam(value = PROTOCOL_PARAM_DESCRIPTION)
```
Comment (ashvayka) [inline application/src/main/java/org/thingsboard/server/controller/DeviceConnectivityController.java:92]: downloadMqttServerCertificate -> downloadServerCertificate

File: ui-ngx/src/assets/locale/locale.constant-en_US.json [hunk 4]
```diff
@@ -1386,7 +1390,24 @@
...
         "unassign-devices-from-edge-title": "Are you sure you want to unassign { count, plural, =1 {1 device} other {# devices} }?",
-        "unassign-devices-from-edge-text": "After the confirmation all selected devices will be unassigned and won't be accessible by the edge."
+        "unassign-devices-from-edge-text": "After the confirmation all selected devices will be unassigned and won't be accessible by the edge.",
+        "time": "Time",
+        "connectivity": {
+            "check-connectivity": "Check connectivity",
+            "device-created-check-connectivity": "Device created. Let's check connectivity!",
+            "loading-check-connectivity-command": "Loading check connectivity commands...",
+            "use-following-instructions": "Use the following instructions for sending telemetry on behalf of the device using shell",
+            "execute-following-command": "Executive the following command",
```
Comment (ViacheslavKlimov) [inline ui-ngx/src/assets/locale/locale.constant-en_US.json:1400]: Execute*

**Pull request merged**